src/main/kotlin/parts/wisdom/arcraider/Arc.kt

```diff
@@ -1,31 +1,34 @@
 package parts.wisdom.arcraider
 
 import java.io.File
+import kotlinx.serialization.*
+import kotlinx.serialization.json.*
 
 val arcData = File(
     checkNotNull(System.getenv("ARC_DATA"),
         { "ARC_DATA environment variable is not set" })
 )
 
+
+@Serializable
 data class Task(
-    val trainingPairs: List<ArcPair>,
-    val testPairs: List<ArcPair>
+    val train: List<ArcPair>,
+    val test: List<ArcPair>
 )
 
+@Serializable
 data class ArcPair(
-    val input: ArcGrid,
-    val output: ArcGrid
+    val input: List<List<Integer>>,
+    val output: List<List<Integer>>
```

Comment:
I like having these as ArcGrid types. It adds a layer of abstraction that makes the properties reflect what the data represents better; even if we have an ArcGrid as just a `typealias`.

Reply:
fixed